.github/workflows/update.yaml

```
@@ -0,0 +1,30 @@
+name: Check for updates
+on:
+  schedule: # for scheduling to work this file must be in the default branch
+  - cron: "0 * * * *" # run every hour
+  workflow_dispatch: # can be manually dispatched under GitHub's "Actions" tab
+
+jobs:
+  flatpak-external-data-checker:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        branch: [ branch/24.08, branch/24.08 ] # list all branches to check
```

```suggestion
        branch: [ branch/24.08 ] # list all branches to check
```
`branch/24.08` must be listed once